src/components/Table/Table.tsx

```diff
@@ -91,20 +90,31 @@ function Table({ problems }: TableProps) {
       </thead>
       <tbody>
         {problems.map((problem) => {
-          const problemIcon = getTaskIcon(problem.completed);
+          const isCompleted = (solvedProblems || []).some(
```

Comment:
`solvedProblems`이 `null`이거나 `undefined`일 수는 없으니까요.

```suggestion
          const isCompleted = solvedProblems.some(
```

Reply:
반영했습니다! 감사합니다!